inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -378,46 +376,42 @@ discretized Gamma.
 
 ## Results
 
-```{r, eval = params$use_epiparameter}
-si_epiparameter <- epiparameter::epidist(
-  params$epiparameter_pathogen,
-  "serial_interval"
+```{r, eval = params$use_epiparameter_database}
+si_epidist <- epiparameter::epidist_db(
+  disease =  params$disease_name,
+  epi_dist =  "serial_interval",
+  single_epidist = TRUE,
+  subset = is_parameterised
```

Comment:
The default is to select distributions that are parameterised over those that are not. Therefore the `subset = is_parameterised` can be removed and the same output should be produced. Please let me know if the output differs after changing this.

Reply:
do you think it's useful to leave it so that when users read the documentation and want to make changes to the template, they are able to see exactly which bit of the code they'd have to change, and easily change is_parameterised for something else, rather than having to read epiparameter's documentation to find which argument to change etc?